LICENSE

```diff
@@ -1,9 +1,18 @@
+Some files are licensed under GPL-3.0-only.  These files have the text
+
+    SPDX-License-Identifier: GPL-3.0-only
+
+in the file header.
```

Comment:
"in the file header.  These files are  Copyright (C) 2015-2020 Maciej Delmanowski <drybjed@gmail.com> and Copyright (C) 2015-2020 DebOps <https://debops.org/>.  See the file `COPYING` for the full text of the GPL-3.0-only license."

Reply:
Hmmm. The file and GPL-3.0-only conflict. (is it ok for them???)
So, we should not include the file. I'm also getting rid of the line which mentions the file.

Reply:
> Hmmm. The file and GPL-3.0-only conflict. (is it ok for them???)

Not sure what you mean.

> So, we should not include the file. I'm also getting rid of the line which mentions the file.

We should not include the `COPYRIGHT` or `COPYRIGHT-debops` file.  We should instead just include the license and copyrights in our `LICENSE` file.